src/datagrid/DataGridValues.tsx

```diff
@@ -387,7 +379,27 @@ export const getPageData = (pageIndex: number, pageSize: number): Promise<DataGr
         }, 2000);
     });
 };
+// Function to get data for page based on pagenumber
+export const getPageDataForSelectedRows = (pageIndex: number, pageSize: number): Promise<DataGridRow[]> => {
+    return new Promise((resolve, reject) => {
+        let rows: DataGridRow[] = [];
 
+        if (pageSize == 5) {
```

Comment:
Whats the suecase here ? Do we at all expect user to provide pageSize to exacely 5,  
also, use === instead of ==

Reply:
@maheshkulkarni4 I have hardcoded this function for pagination story. We are not using it anyware else.